pkgs/development/python-modules/bjoern/default.nix

```diff
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
   checkPhase = ''
     ${python.interpreter} tests/keep-alive-behaviour.py 2>/dev/null
-    ${python.interpreter} tests/test_wsgi_compliance.py
+  '' + stdenv.lib.optionalString isPy3k ''
+    ${python.interpreter} tests/test_wsgi_compliance.py | (! grep -i "fail")
```

Comment:
IMHO, this should be handled upstream. Currently it just print the message https://github.com/jonashaag/bjoern/blob/3.0.0/tests/test_wsgi_compliance.py#L39-L45

Reply:
Absolutely (upstream issue https://github.com/jonashaag/bjoern/issues/158), but in the meantime the way we have the tests setup now, something breaking `test_wsgi_compliance.py` in a more serious way won't get picked up at all.